### PR TITLE
Create CCC_TAC_Minutes-2022-05-19.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

File: TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+#### Voting members
+* Dave Thaler (Microsoft)
+* Zhipeng (Howard) Huang (Huawei)
+* Thomas Fossati (Arm)
+* Dan Middleton (Intel)
+* Giuseppe Giordano (Accenture)
+* Lily Sturmann (Red Hat)
+* Tate Tian (Ant Group)
+
+#### Attendees
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md:56]:
```suggestion
#### Other Attendees
```

File: TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+**Proposed:** That the minutes of the [April 07, 2022](../2022-04-07/TAC_Minutes-2022-04-07.pdf), [April 21, 2022](../2022-04-21/TAC_Minutes-2022-04-21.pdf) and [May 5, 2022](../2022-05-05/TAC_Minutes-2022-05-05.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+ * Quorum was reached and the minutes from April 7 and May 5 were approved. The minutes from April 21 were not approved due to changes being needed.
+ 
+#### Pending approval
+ * [April 21, 2022](../2022-04-21/TAC_Minutes-2022-04-21.pdf)
+
+### TAC Tech Talks
+ *  Logging, debugging and error management in Confidential Computing
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md:81]: Can we 
```suggestion
 *  Logging, debugging and error management in Confidential Computing [slides](Logging%20and%20Confidential%20Computing%20.pdf)
```

File: TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+ *  Presented by - Mike Bursell (MB)
+ *  Lively discussion on architecture and definitions
+ *  DT asked if the TAC should do a white paper or WG, MB - based on blog post, asked for initial blog improvement, Best Practices for next step was propose
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md:84]:
```suggestion
 *  DT asked if the TAC should do a white paper or WG, MB - based on blog post, asked for initial blog improvement, Best Practices for next step was proposed
```
Was anyone assigned to this AI?

File: TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+2. **[Kurt Taylor]** Find out if there is a DCO needed for the Governance repo on GitHub [Issue #107] (https://github.com/confidential-computing/governance/issues/107) (Done)
+
+1. **[Kurt Taylor]** Investigate prior efforts to finalize Graduated lifecycle language, and if needed file an issue to finalize Graduated lifecycle language. (Done)
+   * Issue created to track [Issue #104] (https://github.com/confidential-computing/governance/issues/104)
+
+2. **[Kurt Taylor]** Check if terminology document markdown format will work with LF Creative team and check on their timeline for completing the formatting for the white paper (Done)
+   * Markdown is fine, timeline is 1-3 weeks, depending on length and complexity)
+
+2. **[Kurt Taylor]** Code scanning via LFX Security - add all missing projects to security for scanning, get more details on instructions on installing GitHub bot (Done) 
+    * Projects can set up the bot and repos they want scanned by following these instructions: https://community.lfx.dev/t/how-to-install-and-configure-bots-to-secure-your-projects/181
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md:107]: Can we assign an AI for this?  Or put it on today's agenda?

File: TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+### Project overviews
+### Common Test Infrastructure
+ * DT - status, noted this topic was reviewed with the Governing Board, seeking feedback from the projects, if would be used by the projects, reviewed the budget amounts
+ * DT - main cost is the LF maintainership, not test service or hardware, noted this is not a full-time job from LF
+ * DT proposed that the budget be changed and approved to 50K for LF management, 15K for test hardware
+ * DT PROPOSED: The TAC budget $50k out of the "Cons. IT Services" $100k line item, to allocate to LF management of Common Test Infrastructure, MB asked for clarification on the proposal from the project feedback, DT asked for input from mentors, no additional feedback provided for usage
+ * DM asked for specific written proposal
+
+***APPROVED*** The TAC budget $50k out of the "Cons. IT Services" $100k line item, to allocate to LF management of Common Test Infrastructure
```
Comment (dthaler) [inline TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md:117]: Did anyone have an AI for the next step here?

**Review by dthaler — APPROVED** [review]
Approving but with a few nits that would be good to address and some follow-up questions that we might cover in today's TAC meeting

File: TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md (hunk 5 above)
Comment (dcmiddle) [reply TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md:117]: Maybe we need to bubble out this snippet: "... DT asked for input from mentors..."
We could add a tracked action item for mentors to socialize this with their projects and find out if they have specific needs.

**Review by dcmiddle — APPROVED** [review]

File: TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md (hunk 5 above)
Comment (dcmiddle) [reply TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md:117]: As we ended up discussing next steps during the 2022-06-02 meeting we could pick up further discussion in those minutes.

**Pull request merged**